packages/dx-react-grid-bootstrap3/src/templates/table-select-all-cell.jsx

```diff
@@ -3,9 +3,11 @@ import * as PropTypes from 'prop-types';
 
 export const TableSelectAllCell = ({
   style, allSelected, someSelected, disabled, onToggle,
-  tableColumn, tableRow,
+  tableColumn, tableRow, rowSpan,
   ...restProps
 }) => {
+  const paddingTop = rowSpan > 1 ? (41 * (rowSpan - 1)) : '';
```

Comment:
What does `41` mean? Looks like a magic const. Can we name it somehow?

Reply:
It's a real magic value. I find this number by using simple selection method =/